src/traces/choroplethmapbox/index.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+var deprecationWarning = [
+    '*choroplethmapbox* trace is deprecated!',
+    'Please consider switching to the *choroplethmap* trace type and `map` subplots.'
```

Comment:
@archmoj Maybe I'm out of the loop -- what does the "and `map` subplots" part of this message mean?

Reply:
never mind, I realize now this is referring to `map`, `map2`, etc. in the layout.